backend/package.json

```diff
@@ -25,7 +25,7 @@
     "@types/ramda": "^0.25.40",
     "body-parser": "^1.18.3",
     "cors": "^2.8.4",
-    "elasticsearch": "^14.2.2",
+    "elasticsearch": "^15.4.1",
```

Comment:
Was there no minor version we could upgrade to? I'm a little worried about upgrading majors in the most important library in the whole app.

Reply:
No, it jumps from 14.2.2 to 15.0.0. No vulnerabilities on 15.0.0. We should probably go with 15.0.0 then, right?

https://www.elastic.co/guide/en/elasticsearch/client/javascript-api/current/changelog.html#_15_0_0_may_14_2018

> - Upgrade from lodash 2 to lodash 4 (Thanks @dominykas!) #600
> 
>  - Because this is such a large shift, and it could have potentially subtle effects on the module, we’ve done a major version bump. Unless you are using custom plugins that rely on the old utils, this should be the equivalent of a patch upgrade.

Reply:
No, if 15 is next, let's move to that. This may cause issues.

Reply:
*15.4.1

Reply:
I'm really just worried about the major version.